suite/tests/client-interface/syscall-mod.dll.c

```diff
@@ -48,17 +48,27 @@ dr_emit_flags_t bb_event(void* drcontext, void *tag, instrlist_t* bb,
 {
     instr_t *instr;
     instr_t *next_instr;
-    reg_t in_eax = -1;
+    reg_t in_reg = -1;
 
     for (instr = instrlist_first(bb); instr != NULL; instr = next_instr) {
         next_instr = instr_get_next(instr);
+#ifdef AARCH64
+        if (instr_get_opcode(instr) == OP_movz &&
```

Comment:
Could we make this cross-platform with no ifdefs by using instr_is_mov_constant() here and XINST_CREATE_load_int() below?